usagov/vote-gov-drupal
# VOTE-974 theme audit optimizations

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-974

## Description

Theme optimization based on audit. Remove unused files, optimizing image assets, removing unused variables.

## Deployment and testing

### Post-deploy steps

1. run `npm run build` in votegov theme
2. run `lando drush cr`

### QA/Testing instructions

1. Confirm that compiling the theme results in no errors
2. visit http://vote-gov.lndo.site/ and confirm that the there aren't any regressions related to modified files

## Checklist for the Developer




## Checklist for the Peer Reviewers
